README.md

```diff
@@ -4,67 +4,79 @@
 
-3) Run a pipeline with Caper.
+	> **IMPORTANT**: Make sure that you have python3(> 3.4.1) installed on your system.
 
-## Running pipelines without Caper
+	```bash
+	$ pip install caper  # use pip3 if it doesn't work
+	```
 
-Caper uses the cromwell workflow execution engine to run the workflow on the platform you specify.  While we recommend you use caper, if you want to run cromwell directly without caper you can learn about that [here](docs/deprecated/OLD_METHOD.md).
+3) Read through [Caper's README](https://github.com/ENCODE-DCC/caper) carefully. Find an instruction for your platform. 
+	> **IMPORTANT**: Configure your Caper configuration file `~/.caper/default.conf` correctly for your platform.
 
-## DNAnexus
+## Running a pipeline locally with Caper
 
-You can also run our pipeline on DNAnexus without using Caper or Cromwell. There are two ways to build a workflow on DNAnexus based on our WDL.
+1) Prepare an input JSON file. We will use a subsampled example input JSON based on URLs. Caper will automatically download all fastqs and reference human genome data recursively.
+	```bash
+	$ INPUT_JSON=https://storage.googleapis.com/encode-pipeline-test-samples/encode-atac-seq-pipeline/ENCSR356KRQ_subsampled_caper.json
+	```
 
-1) [dxWDL CLI](docs/tutorial_dx_cli.md)
-2) [DNAnexus Web UI](docs/tutorial_dx_web.md)
+2-1) **Conda**: Run a workflow with Conda. Make sure that you have followed [this instruction](docs/install_conda.md) to install Conda and its environments.
+	> **WARNING**: We no longer recommend Conda for resolving dependencies and plan to phase out Conda support. Instead we recommend using Docker or Singularity. You can install Singularity and use it for our pipeline with Caper (by adding `--singularity` to command line arguments).
 
-## Conda
+	```bash
+	$ conda activate encode-atac-seq-pipeline
+	$ caper run atac.wdl -i ${INPUT_JSON}
+	```
 
-We no longer recommend Conda for resolving dependencies and plan to phase out Conda support. Instead we recommend using Docker or Singularity. You can install Singularity and use it for our pipeline with Caper (by adding `--use-singularity` to command line arguments). Please see [this instruction](docs/install_conda.md).
+2-2) **Singularity (RECOMMENDED)**: Run a workflow with Singularity.
 
-## Tutorial
+	```bash
+	$ caper run atac.wdl -i ${INPUT_JSON} --singularity
+	```
 
-Make sure that you have configured Caper correctly.
-> **WARNING**: Do not run Caper on HPC login nodes. Your jobs can be killed.
+	> **HPCs**: To run multiple workflows on HPCs (e.g. Stanford Sherlock and SCG) see details at [Caper's README](https://github.com/ENCODE-DCC/caper/blob/master/README.md#how-to-run-it-on-slurm-cluster). Do not run Caper on login nodes. Your workflows will get killed. There is a learning curve to understand server/client structure of Caper/Cromwell.
```

Comment:
May be not needed but it might be a good idea to include some resource where the users can try to understand the "There is a learning curve to understand server/client structure of Caper/Cromwell." Like some link that may or may not be too useful but at least they will be able to look at it and learn if they needed help.